elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -86,4 +89,16 @@ default void flush() {
         // default to ignoring criteria
         return loadObjects(entityClass);
     }
+
+    /**
+     * Filter a collection by the Predicates in filterScope.
+     *
+     * @param collection the collection to filter
+     * @param entityClass the class of the entities in the collection
+     * @param predicates the set of Predicate's to filter by
+     * @return the filtered collection
+     */
+    default <T> Collection filterCollection(Collection collection, Class<T> entityClass, Set<Predicate> predicates) {
+        return collection;
```

Comment:
I still feel like we should do something here beyond just returning the collection